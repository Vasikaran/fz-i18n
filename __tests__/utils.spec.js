import {
  pad,
  replaceI18NValuesWithRegex,
  unescapeUnicode,
  getI18NValue
} from '../src/utils';

describe('I18N utils', () => {
  it('should display 01', () => {
    var data = pad('1', 2, '0');
    expect(data).toMatchSnapshot();
  });

  it('should display vimalesan', () => {
    var data = replaceI18NValuesWithRegex('vimal{0}', ['esan']);
    expect(data).toMatchSnapshot();
  });

  it('should display  j', () => {
    var data = unescapeUnicode('\u006A');
    expect(data).toMatchSnapshot();
  });

  it('should display vimal', () => {
    var fn = getI18NValue({ key1: 'vimal' });
    expect(fn('key1')).toMatchSnapshot();
  });
});
