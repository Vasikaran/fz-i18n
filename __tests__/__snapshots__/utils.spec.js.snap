exports[`I18N utils should display  j 1`] = `"j"`;

exports[`I18N utils should display 01 1`] = `"01"`;

exports[`I18N utils should display vimal 1`] = `"vimal"`;

exports[`I18N utils should display vimalesan 1`] = `"vimalesan"`;
